onnxruntime/core/providers/coreml/builders/impl/reshape_op_builder.cc

```diff
@@ -79,7 +79,9 @@ bool ReshapeOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& initializer
     return false;
   }
 
-  const auto& perm_dims = initializers.at(perm_name)->dims();
+  const auto& shape_tensor = *initializers.at(perm_name);
```

Comment:
>shape_tensor [](start = 14, length = 12)

is there a reason that we refer to this initializer as both "perm" and "shape"? it's not so obvious to me

Reply:
It was copied from NNAPI EP, changed both to perm_tensor, and raw_perm